tasks/src/reinforcement-learning/about.md

```diff
@@ -0,0 +1,31 @@
+## Use cases
+
+In this part we will see where we can actually use these algorithms and how it is helping the people in numerous ways.
+
+### Applications in autonomous driving
+
+Autonomous or self driving cars are quite trending now a days. It is also a big step towards driver less commute or where the machine is itself able to perform the task, which makes it an problem of machine learning and artificial intelligence. So the vehicle, driving in the real world without any driver must be backed by an machine learning model trained with all the possibilities. 
+
+In self-driving cars, there are various aspects to consider, such as speed limits at various places, drivable zones, avoiding collisions — just to mention a few. The collection of these variety of situations is tough problem to solve.
+
+Reinforcement learning can be used to prepare a model for this purpose. Reinforcement Learning models are trained in a dynamic environment by learning a policy from its own experiences following the principles of exploration and exploitation that minimize disruption to traffic. Self-driving cars may learn from many aspects which will make it able to make a optimal decision when stuck in a situation.In this case we can also call it as scenario based learning.

```

Comment:
```suggestion
Reinforcement learning can be used to train a model for self-driving cars. Reinforcement Learning models are trained in a dynamic environment by learning a policy from its own experiences, following the principles of exploration and exploitation that minimize disruption to traffic. Self-driving cars can learn from many aspects which will make the car capable of making an optimal decision given a state. In this case we can also call it as scenario based learning.

```

> Reinforcement Learning models are trained in a dynamic environment by learning a policy from its own experiences, following the principles of exploration and exploitation that minimize disruption to traffic
Can you make this part into pieces to ease the readability and explain a bit more about how exploration/exploitation trade-off works in this case to minimize disruption? 

> Self-driving cars can learn from many aspects which will make it able to make a optimal decision when stuck in a situation.In this case we can also call it as scenario based learning.
Aspects of what exactly? 
Can you give a bit of an explanation on scenario based learning too?